avanza/avanza.py

```diff
@@ -1705,3 +1711,112 @@ def create_monthly_saving(
+                E.g. it is fully optional to provide account_id and/or transaction_type.
+
+            transactions_from: Fetch transactions from this date.
+
+            transactions_to: Fetch transactions to this date.
+
+            min_amount: Only fetch transactions of at least this value.
+
+            max_amount: Only fetch transactions of at most this value.
+
+            order_book_ids: Only fetch transactions involving this/these orderbooks.
+
+        Returns:
+            {
+                'transactions': [
+                    {
+                        'account': {
+                            'type': str,
+                            'name': str,
+                            'id': int
+                        },
+                        'noteId': str,
+                        'transactionType': str,
+                        'verificationDate': str,
+                        'sum': float,
+                        'description': str,
+                        'currency': str,
+                        'amount': float,
+                        'orderbook': {
+                            'isin': str,
+                            'currency': str,
+                            'name': str,
+                            'id': int,
+                            'type': str
+                        },
+                        'price': float,
+                        'volume': float,
+                        'id': str
+                    },
+                ],
+                'totalNumberOfTransactions': int,
+                'totalAmounts': {
+                    str: {
+                        'total': float,
+                        'BUY': {
+                            'total': float,
+                            'orderbooks': [
+                                {
+                                    'total': float,
+                                    'isin': str,
+                                    'currency': str,
+                                    'name': str,
+                                    'flagCode': str,
+                                    'id': int,
+                                    'type': str
+                                },
+                            ]
+                        },
+                        'SELL': {
+                            'total': float,
+                            'orderbooks': [
+                                {
+                                    'total': float,
+                                    'isin': str,
+                                    'currency': str,
+                                    'name': str,
+                                    'flagCode': str,
+                                    'id': int,
+                                    'type': str
+                                },
+                            ]
+                        }
+                    }
+                }
+            }
+        """
+        options = {
```

Comment:
Since transaction_from and transcation_to are optional .isoformat() raises an error if not supplied.
I guess you'll have to do it the same way as min_amount/max_amount and add them conditionally.

Reply:
Ah, nice catch. Done now.